### PR TITLE
Logic for non-dash

### PR DESCRIPTION
**Pull request by ives1227**
Added in logic to skip 'non-DASH' items

JIRA: https://at-harvard.atlassian.net/browse/ETD-362

Tested in dev

### CODE REVIEW

File: etd/worker.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ class Worker():
     aipPattern = '.+_(\\d+).zip'
     reAipPackage = re.compile(aipPattern)
     datePattern = '\\d{5}(-\\d\\d-\\d\\d)'
-    re5digitDate = re.compile(datePattern)
+    re4digitDate = re.compile(datePattern)
```
Comment (cvicary) [inline etd/worker.py:56]: Did you mean to revert the name on this, considering that `datePattern` is for a 5-digit year?

**Review by cvicary — COMMENTED** [review]
All the logic looks right. I just had a minor question related to naming of the `re4digitDate` variable.

**Comment by awoods** [comment]
Is there a Jira for this? It seems like an important update that we want to track.

File: etd/worker.py (hunk 1 above)
Comment (ives1227) [reply etd/worker.py]: Fixed

**Comment by ives1227** [comment]
> Is there a Jira for this? It seems like an important update that we want to track.

Added

**Review by michael-lts — APPROVED** [review]
Looks good

**Pull request merged**